python/src/aiconfig/scripts/aiconfig_cli.py

```diff
@@ -100,7 +123,7 @@ def _run_editor_servers(edit_config: EditServerConfig) -> Result[list[str], str]
             return Err(e)
 
     results: list[Result[str, str]] = []
-    backend_res = run_backend_server(edit_config)
+    backend_res = run_backend_server(edit_config, cli_config_path)
```

Comment:
nit: I really do not like using `res`. Can we pls use `response` or `result` so it's clear which it is?

Reply:
Sure, actually we should pick a convention that doesn't do the "maybe_..." thing that @rholinshead pointed out.

One suggestion:

```
backend_server_outcome = ... 
match backend_server_outcome:
    Ok(backend_server_outcome_ok)
        ... 
    Err(e)
        ... 

```